styles/GlobalStyles.tsx

```diff
@@ -125,6 +125,100 @@ export default createGlobalStyle`
         z-index: 99;
     }
 
+    .bn-onboard-modal-terms-of-service {
+        padding: 0 1rem;
+    }
+
+    /** Description Styles */
+    .bn-onboard-select-description {
+        padding: 0 1rem;
+    }
+
+    /** Header Styles */
+    .bn-onboard-modal-content-header {
+        padding: 0 1rem;
+    }
+
+    .bn-onboard-modal-content-header-icon {
+        display: none!important;
+    }
+
+    .bn-onboard-modal-content-header h3 {
+        color: var(--color-text)!important;
+        font-weight: lighter!important;
+        margin-left: 0!important;
+    }
+
+    /** Wallet Styles */
+
+    /** Legal modal*/
+    #legal-modal {
+        max-width: 420px;
+    }
+    #legal-modal #checkbox-container {
+        display: flex;
+        margin-top: 16px;
+    }
+
+    #legal-modal span,
+    #legal-modal input {
+        color: #fff;
+    }
```

Comment:
Try to avoid throwing stuff in Global styles when it can just be put in the component style of the individual component. Since legal modal styles are only used on Legal modal, it makes sense to put them in the individual component styles